examples/analyse_counterpoise.py

```diff
@@ -1,31 +1,43 @@
+#!/usr/bin/env python3
 '''
 
 TODO: Needs high level description
```

Comment:
The description is required, but technically an issue outside of the goal of this pull request.